### PR TITLE
feat(ui): adding link to event page from chapter dashboard

### LINKED ISSUE #1941: Chapter dashboard - links to events
When in specific chapter dashboard (`/dashboard/chapter/{id}`), there's list with events from this chapter. Links to the events are leading to public event page (`/events/{id}`).

There's no link to the event dashboard page (`/dashboard/events/{id}`). This makes navigating to certain event, found on the chapter's event list, slightly inconvenient, as it's needed to go to events dashboard (`/dashboard/events`) and find wanted event on another list.

Simplest would be replacing link to lead to the event dashboard. Better might be including both links, with visual, or otherwise, indication which link is to event dashboard and which to event page.

### PR DESCRIPTION
**Pull request by sneha-afk**
Co-authored by: [dianachu18](https://github.com/dianachu18) and [jamesrcramos](https://github.com/jamesrcramos)

- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Closes #1941

Events have two pages, their dashboard and their public page. Previously, clicking on the event name within the chapter dashboard lead to the event dashboard (`/dashboard/events/${id}`). The event name is now plain text, and two buttons have been added: one to lead to the dashboard (`/dashboard/events/${id}`) and another to lead to the public page (`/events/${id}`). The page has been re-formatted to use a `Grid` component instead of a `Flex` component.

The page containing these changes is:
> /dashboard/chapters/${id}

The file containing these changes is:
> client/src/modules/dashboard/shared/components/EventList.tsx

### CODE REVIEW

File: client/src/modules/dashboard/shared/components/EventList.tsx [hunk 1]
```diff
@@ -26,38 +34,54 @@ export const EventList = ({ events, emptyText, title }: Props) => {
       {events.length > 0 ? (
         <Grid gap="2em">
           {events.map(({ canceled, id, invite_only, name }) => (
-            <Flex justifyContent="space-between" key={id}>
-              <LinkButton href={`/dashboard/events/${id}`}>{name}</LinkButton>
```
Comment (Sboonny) [inline client/src/modules/dashboard/shared/components/EventList.tsx:30]: Using grid doesn't provide the responsive layout desired from grid system,

Reading the issue, you can create something similar to the menu in the header for this button, here is the chakra docs 

https://chakra-ui.com/docs/components/menu#menuitem-as-a-link

Comment (gikf) [reply client/src/modules/dashboard/shared/components/EventList.tsx:30]: @Sboonny I don't have anything specific in mind

Comment (sneha-afk) [reply client/src/modules/dashboard/shared/components/EventList.tsx:30]: Thank you for the suggestions, we will adjust our implementation.

**Review by Sboonny — COMMENTED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @sneha-afk, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Review by Sboonny — APPROVED** [review]

**Comment by gikf** [comment]
Hey @sneha-afk,

could you confirm how many people were involved in this PR as a first-time contributors and are needed to be added to the contributors?

**Comment by sneha-afk** [comment]
Hello @gikf , we worked as a group of 3: myself, @dianachu18 and @jamesrcramos. Thank you!

**Comment by gikf** [comment]
@all-contributors please be so kind and: add @dianachu18 for code, add @jamesrcramos for code, add @sneha-afk for code.

**Comment by gikf** [comment]
Welp

@all-contributors please add @jamesrcramos for code, please add @sneha-afk for code.

**Comment by gikf** [comment]
@all-contributors please add @sneha-afk for code.

**Pull request merged**